### DISCUSSION TITLE
Not a fan of tab to skip

### DISCUSSION THREAD

**Discussion by yash-fn** [aradzie/keybr.com#49] [Ideas]
pressing tab seems to skip to the next word. I am not a fan of this because I hit tab by accident all the time, such as when practicing the letter q.

**Comment by aradzie** [comment]
It's because all white-space keys, such as Tab and Return have the same effect as the Space key.

https://github.com/aradzie/keybr.com/blob/master/packages/keybr-textinput/lib/normalize.ts#L35

Today I added a new typing assist, "Space skips words". When you hit Tab, it also skips words for the reason I explained above.

Maybe I should leave Tab as Tab and Enter as Enter without converting them to Space.

**Comment by yash-fn** [comment]
Can't speak for others, but unlike space and enter, which I almost never hit accidentally, I seem to always keep hitting tab by missing q. but either way disabling the space to skip option fixed this issue altogether for me since I never really needed a reason to skip a word anyways. Thanks for the guidance!
